src/DurableTask.Core/Query/OrchestrationStatusQueryCondition.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
For new or updated source code in this repo, I like to have us enable C# nullable reference types: https://docs.microsoft.com/en-us/dotnet/csharp/nullable-references. It's similar in spirit to `@Nullable` and `@NonNull` in Java, but more directly integrated into the C# compiler. You can enable it on this file by adding the following line:
```suggestion
#nullable enable
```
Note that it will require you to add `?` to the end of all the property types that are reference types (e.g. string, ICollection<T>, etc.) since these properties are not initialized in the constructor.